clang/test/BoundsSafety/Frontend/ignored_for_asm.s

```diff
@@ -0,0 +1,5 @@
+
+// RUN: %clang -fbounds-safety-experimental -fsyntax-only %s 2>&1 | FileCheck %s
+// RUN: %clang_cc1 -fbounds-safety-experimental -fsyntax-only %s 2>&1 | FileCheck %s
+
+// CHECK: warning: '-fbounds-safety' is ignored for assembly
```

Comment:
@rapidsna  Is some code missing from this PR? I see you now have a test but I can't find the code change that causes this warning to be emitted in the PR.

Reply:
Done!